sdk/inc/azure/iot/az_iot_common.h

```diff
@@ -164,7 +164,9 @@ AZ_NODISCARD az_result az_iot_message_properties_find(
  * @param[in] properties The #az_iot_message_properties to use for this call
  * @param[out] out_name A pointer to an #az_span containing the name of the next property.
  * @param[out] out_value A pointer to an #az_span containing the value of the next property.
- * @return #az_result
+ * @return An #az_result value indicating the result of the operation.
+ *   @retval #AZ_OK A property was retrieved successfully.
+ *   @retval #AZ_ERROR_IOT_NO_MORE_PROPERTIES The API reached the end of the properties to retrieve.
```

Comment:
```suggestion
 * @return An #az_result value indicating the result of the operation.
 * @retval #AZ_OK A property was retrieved successfully.
 * @retval #AZ_ERROR_IOT_NO_MORE_PROPERTIES The API reached the end of the properties to retrieve.
```

Reply:
I like it offset to show the diff but okayyyyyy I'll abide by consistency

Reply:
I am so used to seeing `yyyy` as part of datetime formats (particularly datetime **offset** in .NET), I had to do a double take at your comment, and wondered, what does yyyy-mm-dd have to do with `az_result`?